modules/dreamview/frontend/src/renderer/tileground.js

```diff
@@ -82,28 +77,15 @@ export default class TileGround {
             if (i !== row || j !== col) {
                 return;
             }
-            const itemPos = coordinates.applyOffset({
-                x: item.x,
-                y: item.y,
-                z: 0,
-            });
-            loadObject(material, object, scales,
-                mesh => {
-                    mesh.rotation.x = Math.PI / 2;
-                    mesh.rotation.y = item.heading;
-                    mesh.position.set(itemPos.x, itemPos.y, itemPos.z);
-                    if (item.id) {
-                        // store signal id as name in mesh
-                        mesh.name = item.id;
-                    }
-                    mesh.matrixAutoUpdate = false;
-                    mesh.updateMatrix();
+            item.type = type;
```

Comment:
Why store a type here given we are passing a type to addTrafficControl? Overall not a fan of passing this `type` around, would rather have function like addTrafficSignal and addStopSign.

Reply:
Separate traffic light and stop sign now